project/driver_controls/inc/driver_state.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Why is this called `driver_state.h`? It doesn't seem to have anything to do with the driver's state.